packages/components/src/List/ListComposition/Manager/ListManager.component.js

```diff
@@ -41,6 +42,7 @@ Manager.defaultProps = {
 	t: getDefaultT(),
 };
 Manager.propTypes = {
+	id: PropTypes.string,
```

Comment:
If it's not mandatory, what will happen if it's not set?

Reply:
Good point, now it's required